plugins/linuxplugin/common/model/interfaces/interfaces.proto

```diff
@@ -3,42 +3,41 @@ syntax = "proto3";
 package interfaces;
 
 message LinuxInterfaces {
-    enum InterfaceType {
+    enum InterfaceType {                    /* Available interface types */
         VETH = 0;
+        AUTO_TAP = 1;
     };
 
     message Interface {
-        string name = 1;
+        string name = 1;                    /* Unique name of the interface used the etcd key */
         string description = 2;
         InterfaceType type = 3;
         bool enabled = 4;
-        string phys_address = 5;
-        uint32 mtu = 6;
-        string host_if_name = 7; /* name of the interface in the host OS */
+        repeated string ip_addresses = 5;   /* IP address in format <ipAddress>/<ipPrefix> */
+        string phys_address = 6;            /* MAC address */
+        uint32 mtu = 7;                     /* Maximum transmission unit value */
+        string host_if_name = 8;            /* Name of the interface in the host OS. If not set, the host name
+                                               is the same as interface name */
```

Comment:
please extend the model in order to cover use-case where we want to
 find an interface identified by `tmp_host_if_name`  then move it to a namespace and rename it to the `final_host_if_name`

Maybe something like
```protobuf
message AutoTap {
   string tmp_host_if_name = 1;
}
```